src/app/account-app/account-upgrades.component.html

```diff
@@ -62,55 +68,50 @@ <h2> Main account subscriptions </h2>
     </p>
 
     <div *ngIf="limitedTimeOffer" class="limitedTimeOffer">
-      <h2 style="margin: 20px 0 10px 0"><mat-icon class="limitedTimeOffer">star</mat-icon> Limited time offer <mat-icon class="limitedTimeOffer">star</mat-icon></h2>
+      <h2 style="margin: 20px 0 10px 0">
+        <mat-icon class="limitedTimeOffer">star</mat-icon> Limited time offer <mat-icon class="limitedTimeOffer">star
+        </mat-icon>
+      </h2>
       <h3 style="margin: 10px 0; font-weight: normal;">We're feeling extra generous today!</h3>
       <h4 style="margin: 20px 0 0 0">Subscribe within 24 hours of signing up and get</h4>
       <h1 style="margin: 10px 0 0 0">1 extra year for free!</h1>
-      <p>This offer expires in <app-runbox-timer class="timer" [user_created]="me.getCreatedMoment()" [timer_length]="limited_time_offer_age" (onFinished)="runboxTimerFinished()"></app-runbox-timer></p>
+      <p>This offer expires in <app-runbox-timer class="timer" [user_created]="me.getCreatedMoment()"
+          [timer_length]="limited_time_offer_age" (onFinished)="runboxTimerFinished()"></app-runbox-timer>
+      </p>
     </div>
 
     <div id="productGrid" *ngIf="subs_regular | async as subs_regular; else productsLoading">
-      <app-account-product
-        *ngFor="let p of subs_regular"
-        [ngClass]="'productCard'"
-        [p]="p"
-        [currency]="me.currency"
-        >
+      <app-account-product *ngFor="let p of subs_regular" [ngClass]="'productCard'" [p]="p" [currency]="me.currency">
         <ul>
           <li *ngFor="let detail of p.details"> {{ detail }}
         </ul>
       </app-account-product>
     </div>
 
     <div id="productGrid" *ngIf="subs_special | async as subs_special; else productsLoading">
-      <app-account-product
-        *ngFor="let p of subs_special"
-        [ngClass]="'productCardSpecial'"
-        [p]="p"
-        [currency]="me.currency"
-        >
+      <app-account-product *ngFor="let p of subs_special" [ngClass]="'productCardSpecial'" [p]="p"
```

Comment:
I'm not sure if this is an improvement, but alright :P

What formatter did you use?

Reply:
Maybe i should make separate PRs for this formatting stuff. 
All I did is wrap all the pages in <section class="mat-typography"> which changes the font styles and formatting to more minimal.

Reply:
Then I also had to change some of the headings from `h4` to `h1` to make it look the same across all the pages. 
Adding the wrapper automatically indented all the rest of the code though so that's a boomer. Wanted to avoid it but the files would look ugly.

Reply:
Rightright. No harm done :)